src/gpuhunt/providers/vultr.py

```diff
@@ -0,0 +1,182 @@
+            gpu_memory=None,
+            gpu_vendor=None,
+            spot=False,
+            disk_size=plan["disk"],
+        )
+    elif plan_type == "vcg":
+        gpu_name = plan["gpu_type"].split("_")[1] if "_" in plan["gpu_type"] else None
+        if gpu_name in EXCLUSION_LIST:
+            logger.info(f"Excluding plan with GPU {gpu_name} as it is not supported.")
+            return None
+        gpu_vendor = get_gpu_vendor(gpu_name)
+        gpu_memory_gb = plan["gpu_vram_gb"]
+        gpu_count = (
+            max(1, gpu_memory_gb // get_gpu_memory(gpu_name)) if gpu_name else 0
+        )  # For fractional GPU,
+        # gpu_count=1
+        return RawCatalogItem(
+            instance_name=plan["id"],
+            location=location,
+            price=plan["hourly_cost"],
+            cpu=plan["vcpu_count"],
+            memory=plan["ram"] / 1024,
+            gpu_count=gpu_count,
+            gpu_name=gpu_name,
+            gpu_memory=gpu_memory_gb / gpu_count,
+            gpu_vendor=gpu_vendor,
+            spot=False,
+            disk_size=plan["disk"],
+        )
+
+
+def get_gpu_memory(gpu_name: str) -> float:
+    if gpu_name.upper() == "A100":
+        return 80  # VULTR A100 instances have 80GB
+    for gpu in KNOWN_NVIDIA_GPUS:
+        if gpu.name.upper() == gpu_name.upper():
+            return gpu.memory
+
+    for gpu in KNOWN_AMD_GPUS:
+        if gpu.name.upper() == gpu_name.upper():
+            return gpu.memory
+    logger.warning(f"Unknown GPU {gpu_name}")
+
+
+def get_gpu_vendor(gpu_name: Optional[str]) -> Optional[str]:
+    if gpu_name is None:
+        return None
+    for gpu in KNOWN_NVIDIA_GPUS:
+        if gpu.name.upper() == gpu_name.upper():
+            return AcceleratorVendor.NVIDIA.value
+    for gpu in KNOWN_AMD_GPUS:
+        if gpu.name.upper() == gpu_name.upper():
+            return AcceleratorVendor.AMD.value
+    return None
+
+
+def extract_gpu_info_from_id(id_str: str):
+    parts = id_str.split("-")
+    if "gpu" in parts:
+        gpu_name = parts[-2].upper()
+        try:
+            gpu_count = int(parts[-3])
+        except ValueError:
+            gpu_count = 1  # Default set to 1 if count is not explicitly specified,
+            # for instance in vbm-64c-2048gb-l40-gpu count is not specified but
+            # in vbm-64c-2048gb-8-l40-gpu count is specified as 8
+        return gpu_name, gpu_count
+    return None, 0
+
+
+def _make_request(method: str, path: str, data: Any = None) -> Response:
+    response = requests.request(
+        method=method,
+        url=API_URL + path,
+        json=data,
+        timeout=30,
+    )
+    response.raise_for_status()
+    return response
+
+
+BARE_METAL_GPU_DETAILS = {
+    "vbm-48c-1024gb-4-a100-gpu": (4, "A100", 80),
+    "vbm-112c-2048gb-8-h100-gpu": (8, "H100", 80),
+    "vbm-112c-2048gb-8-a100-gpu": (8, "A100", 80),
+    "vbm-64c-2048gb-8-l40-gpu": (8, "L40S", 48),
+    "vbm-72c-480gb-gh200-gpu": (1, "GH200", 480),
```

Comment:
![Screenshot From 2025-01-08 12-03-18](https://github.com/user-attachments/assets/43578b7c-864f-441b-8d71-85688ac35d36)
```suggestion
    "vbm-72c-480gb-gh200-gpu": (1, "GH200", 96),
```